packages/adapter-static/package.json

```
@@ -29,9 +29,11 @@
 	},
 	"devDependencies": {
 		"@sveltejs/kit": "workspace:*",
+		"cookie": "^0.4.1",
```

https://github.com/sveltejs/kit/pull/4592
```suggestion
		"cookie": "^0.5.0",
```